configs/products.json

```
@@ -1,122 +1,242 @@
+        "supported_edition": ["kde"]
     },
     {
         "product": "rock-4c-plus",
-        "soc": "rk3399t"
+        "product_name": "ROCK 4C+",
+        "product_full_name": "ROCK 4C+",
+        "soc": "rk3399t",
+        "supported_suite": ["bullseye"],
+        "supported_edition": ["kde"]
     },
     {
         "product": "rock-4se",
-        "soc": "rk3399t"
+        "product_name": "ROCK 4SE",
+        "product_full_name": "ROCK 4SE",
+        "soc": "rk3399t",
+        "supported_suite": ["bullseye"],
+        "supported_edition": ["kde"]
     },
     {
         "product": "rock-pi-4a",
-        "soc": "rk3399"
+        "product_name": "ROCK Pi 4A",
+        "product_full_name": "ROCK Pi 4A",
+        "soc": "rk3399",
+        "supported_suite": ["bullseye"],
+        "supported_edition": ["kde"]
     },
     {
         "product": "rock-pi-4b",
-        "soc": "rk3399"
+        "product_name": "ROCK Pi 4B",
+        "product_full_name": "ROCK Pi 4B",
+        "soc": "rk3399",
+        "supported_suite": ["bullseye"],
+        "supported_edition": ["kde"]
     },
     {
         "product": "rock-pi-4c",
-        "soc": "rk3399"
+        "product_name": "ROCK Pi 4C",
+        "product_full_name": "ROCK Pi 4C",
+        "soc": "rk3399",
+        "supported_suite": ["bullseye"],
+        "supported_edition": ["kde"]
     },
     {
         "product": "rock-pi-4a-plus",
-        "soc": "op1"
+        "product_name": "ROCK Pi 4A+",
+        "product_full_name": "ROCK Pi 4A+",
+        "soc": "op1",
+        "supported_suite": ["bullseye"],
+        "supported_edition": ["kde"]
     },
     {
         "product": "rock-pi-4b-plus",
-        "soc": "op1"
+        "product_name": "ROCK Pi 4B+",
+        "product_full_name": "ROCK Pi 4B+",
+        "soc": "op1",
+        "supported_suite": ["bullseye"],
+        "supported_edition": ["kde"]
     },
     {
         "product": "rock-pi-n10",
-        "soc": "rk3399pro"
+        "product_name": "ROCK Pi N10",
+        "product_full_name": "ROCK Pi N10",
+        "soc": "rk3399pro",
+        "supported_suite": ["bullseye"],
+        "supported_edition": ["kde"]
     },
     {
         "product": "radxa-cm3-io",
-        "soc": "rk3566"
+        "product_name": "Radxa CM3 IO",
+        "product_full_name": "Radxa ROCK 3 Compute Module IO Board",
+        "soc": "rk3566",
+        "supported_suite": ["bullseye"],
+        "supported_edition": ["xfce"]
     },
     {
         "product": "radxa-cm3-rpi-cm4-io",
-        "soc": "rk3566"
+        "product_name": "Radxa CM3 RPI CM4 IO",
+        "product_full_name": "Radxa ROCK 3 Compute Module with Raspberry Pi Compute Module 4 IO Board",
+        "soc": "rk3566",
+        "supported_suite": ["bullseye"],
+        "supported_edition": ["xfce"]
     },
     {
         "product": "radxa-cm3-sodimm-io",
-        "soc": "rk3566"
+        "product_name": "Radxa CM3 SODIMM IO",
+        "product_full_name": "Radxa ROCK 3 Compute Module SODIMM IO Board",
```

```suggestion
        "product_name": "Radxa CM3S IO",
        "product_full_name": "Radxa CM3S IO Board",
```
现在重命名成cm3s，不带SODIMM了